models.py

```diff
@@ -603,3 +671,29 @@ def Store(cls, institution, session_name, errorcheck_data):
     errorcheck = ErrorCheck(data = errorcheck_data)
     errorcheck.key = ErrorCheck.errorcheck_key(institution, session_name)
     errorcheck.put()
+
+
+class DBVersion(ndb.Model):
+  data = ndb.IntegerProperty()
+
+  @classmethod
+  @timed
+  def db_version_key(cls, institution, session):
+    return ndb.Key("InstitutionKey", institution,
+                   Session, session,
+                   DBVersion, "db_version")
+  @classmethod
+  @timed
+  def Fetch(cls, institution, session):
+    db_version = DBVersion.db_version_key(institution, session).get()
+    if db_version:
+      return db_version.data
+    else:
+      return 0
+
+  @classmethod
+  @timed
+  def Store(cls, institution, session, version):
```

Comment:
If you create a new session, does it complain that the db needs upgrade? It probably shouldn't. We could fix that by storing the current db version whenever we create a new session.